src/widgets/BasicMenu.cpp

```diff
@@ -199,7 +199,9 @@ void ReplayPopup( wxMenu *theMenu )
    }
 
    // Replay did not find all as expected
-   throw Journal::SyncException{};
+   throw Journal::SyncException(wxString::Format(
+      "PopuMenu has failed to invoke %s",
+      wxJoin(fields, ',').ToStdString().c_str()));
```

Comment:
I think `wxJoin(fields, '.')` is all youy need

Reply:
wxWidgets uses C va_format, it can only be used with `const char*`

Reply:
Oho yes.  I had `TranslatableString` on my mind.